{{ cookiecutter.__project_name_kebab }}/{{ cookiecutter.__project_name_snake }}/static_src/custom.d.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I put this here because if I just make a new package and run eslint on the output, I get

```
  30:9   error  'wagtailConfig' is defined but never used  no-unused-vars
  35:12  error  'gettext' is defined but never used        no-unused-vars
  35:20  error  'text' is defined but never used           no-unused-vars
  38:12  error  'ngettext' is defined but never used       no-unused-vars
  38:21  error  'singular' is defined but never used       no-unused-vars
  38:39  error  'plural' is defined but never used         no-unused-vars
  38:55  error  'count' is defined but never used          no-unused-vars
  60:12  error  'get_format' is defined but never used     no-unused-vars
  60:23  error  'formatType' is defined but never used     no-unused-vars
  63:12  error  'gettext_noop' is defined but never used   no-unused-vars
  63:25  error  'text' is defined but never used           no-unused-vars
  66:12  error  'pgettext' is defined but never used       no-unused-vars
  66:21  error  'context' is defined but never used        no-unused-vars
  66:38  error  'text' is defined but never used           no-unused-vars
  69:21  error  'context' is defined but never used        no-unused-vars
  69:38  error  'text' is defined but never used           no-unused-vars
  69:52  error  'count' is defined but never used          no-unused-vars
  72:12  error  'pluralidx' is defined but never used      no-unused-vars
  72:22  error  'count' is defined but never used          no-unused-vars

✖ 19 problems (19 errors, 0 warnings)
```

This is not unexpected as none of these type definitions are used by any code that exists (yet). I think if we want to be able to run eslint on this and have it pass, we do need to silence that error somehow.

I am slightly unclear whether we want to render the package output with that check disabled when a user generates a new package. I think I need to understand slightly more about the rationale for including this specific selection of type definitions in the first place to make the right decision here.